### PR TITLE
Fix non-deterministic sorting

### PR DESCRIPTION
**Pull request by gigxz**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Add tie-breaker sorts on `id` to fix presenting issues with nondeterministic sorting

## Type of change
- [x] Bug fix

## Checklist before requesting review
- [ ] I have performed a self-review of my code
- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [ ] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [ ] My code follows the style guidelines of this project (rubocop)
- [ ] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis/app/models/hmis/hud/household.rb [hunk 1]
```diff
@@ -113,7 +113,9 @@ def self.sort_by_option(option)
         hh_t[:any_wip].eq(true).desc,
         hh_t[:latest_exit].eq(nil).desc,
         earliest_entry: :desc,
+        id: :desc,
       )
+    # FIXME these HoH sort orders aren't working as desired, they sort based on all household members
```
Comment (gigxz) [inline drivers/hmis/app/models/hmis/hud/household.rb:118]: @martha I found a bug while reviewing these, it looks like the sort is actually based on all household members, not just the HoH. I think it's low prio to fix, but would you mind adding a ticket to the backlog? thank you!

Screenshot 2024-05-01 at 8 53 40 AM

**Review by ttoomey — APPROVED** [review]
Looks great!

File: drivers/hmis/app/models/hmis/hud/household.rb (hunk 1 above)
Comment (martha) [reply drivers/hmis/app/models/hmis/hud/household.rb:118]: Yes, sorry about that! Created https://github.com/open-path/Green-River/issues/6002

**Review by martha — APPROVED** [review]

File: drivers/hmis/app/models/hmis/hud/household.rb (hunk 1 above)
Comment (gigxz) [reply drivers/hmis/app/models/hmis/hud/household.rb:118]: thanks!

**Pull request merged**